collateral-onboarding/hardhat/Dockerfile

```diff
@@ -0,0 +1,27 @@
+ARG NODE_VERSION=node:16.9.1
+
+FROM $NODE_VERSION AS dependency-base
+
+# create destination directory
+RUN mkdir -p /hardhat
+
+# copy the core, note .dockerignore
+WORKDIR /hardhat
+COPY ./package.json .
+COPY ./package-lock.json .
+
+COPY ./ /hardhat
+
+RUN npm ci
+
+FROM $NODE_VERSION AS development
+
+ARG ALCHEMY_URL
+ENV ALCHEMY_URL=${ALCHEMY_URL}
```

Comment:
Please provide rood readme with instructions on how to start, the list of required env vars, and any other magic required (eg M1-related)